pkg/cosign/tlog.go

```diff
@@ -138,6 +138,27 @@ func GetRekorPubs(ctx context.Context, _ *client.Rekor) (map[string]RekorPubKey,
 	return publicKeys, nil
 }
 
+// rekorPubsFromClient returns a RekorPubKey keyed by the log ID from the Rekor client.
+// NOTE: This should **not** be used in the verification path, but may be used in the
+// sign path to validate return responses are consistent from Rekor.
+func rekorPubsFromClient(rekorClient *client.Rekor) (map[string]RekorPubKey, error) {
+	publicKeys := make(map[string]RekorPubKey)
+	pubOK, err := rekorClient.Pubkey.GetPublicKey(nil)
+	if err != nil {
+		return nil, fmt.Errorf("unable to fetch rekor public key from rekor: %w", err)
+	}
+	pubFromAPI, err := PemToECDSAKey([]byte(pubOK.Payload))
```

Comment:
this is just like L103-111, wondering if that could be hoisted out to a function that we can tell call from here (and hopefully remove the ENV stuff from GetRekorPubs but do the equivalent call to it from 'cli' portion instead). I assume at least that would be desirable?
We can do it in a follow up, but just curious if that even makes sense?

Reply:
addressed in commit, but only to factor out the logic and make it easier to construct a map of trusted rekor pub keys outside TUF